components/views/auth/ResetPassword.tsx

```diff
@@ -60,8 +138,8 @@ const ResetPassword = () => {
                         </FormRow>
                         <SubmitButton
                             label="Submit"
-                            disabled={!isFormValid}
-                            loading={false}
+                            disabled={disableSubmit}
```

Comment:
This still doesn't have the loading check baked in with the disable

Reply:
we talked